experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/detail.html

```diff
@@ -1,14 +1,365 @@
+            </tr>
+            <tr>
+              <th>Feature config</th>
+              <td>
+                {% for feature in experiment.feature_configs.all %}
+                  {{ feature.name }} - {{ feature.description }}
+                {% empty %}
+                  <span class="text-danger">Not set</span>
+                {% endfor %}
+              </td>
+              <th>Advanced targeting</th>
+              <td>{{ experiment.targeting_config.name }} - {{ experiment.targeting_config.description }}</td>
+            </tr>
+            <tr>
+              <th>Hypothesis</th>
+              <td colspan="3">{{ experiment.hypothesis }}</td>
+            </tr>
+            <tr>
+              <th>Primary Outcomes</th>
+              <td colspan="3">
+                {% for outcome, url in primary_outcome_links %}
+                  <a href="{{ url }}" target="_blank" rel="noopener noreferrer">{{ outcome.title|remove_underscores }}</a>
+                  {% if not forloop.last %},{% endif %}
+                {% empty %}
+                  <span class="text-danger">Not set</span>
+                {% endfor %}
+              </td>
+            </tr>
+            <tr>
+              <th>Secondary Outcomes</th>
+              <td colspan="3">
+                {% for outcome, url in secondary_outcome_links %}
+                  <a href="{{ url }}" target="_blank" rel="noopener noreferrer">{{ outcome.title|remove_underscores }}</a>
+                  {% if not forloop.last %},{% endif %}
+                {% empty %}
+                  <span class="text-danger">Not set</span>
+                {% endfor %}
+              </td>
+            </tr>
+            <tr>
+              <th>Team projects</th>
+              <td>
+                {% for project in experiment.projects.all %}
+                  <p>{{ project }}</p>
+                {% empty %}
+                  <span class="text-danger">Not set</span>
+                {% endfor %}
+              </td>
+              <th>Subscribers</th>
+              <td>
+                {% for subscriber in experiment.subscribers.all %}
+                  <p>{{ subscriber }}</p>
+                {% empty %}
+                  <span class="text-danger">Not set</span>
+                {% endfor %}
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <!-- Risk Mitigation Questions Card -->
+    <div class="card mb-3">
+      <div class="card-header">
+        <h4>Risk Mitigation Questions</h4>
+      </div>
+      <div class="card-body">
+        <table class="table table-striped">
+          <tbody>
+            <tr>
+              <th colspan="3">{{ RISK_QUESTIONS.BRAND }}</th>
+              <td class="{% if experiment.risk_brand %}text-danger font-weight-bold{% endif %}">
+                {% if experiment.risk_brand != None %}
+                  {{ experiment.risk_brand|yesno }}
```

Comment:
Oh it's good that these `yesno` are a simple filter, but they come out as lower case and they should be upper case.  We can pass it in as `yesno:"Yes,No"`.